### ISSUE TITLE
Unable to perform sha256sum on v12.9.1

### ISSUE THREAD

**Issue by longthanhtran** [asdf-vm/asdf-nodejs#139]
As I am new to asdf-nodejs and followed the instructions to install the plugin to replace old nvm, even after I do `import-release-team-keyring` but the plugin keeps telling me it unable to verify checksum, log below

```zsh
» asdf install nodejs 12.9.1           
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4003    0  4003    0     0   9530      0 --:--:-- --:--:-- --:--:--  9530
gpg: Signature made Tue 27 Aug 2019 01:37:22 AM +07
gpg:                using RSA key 8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600
gpg: Good signature from "Michaël Zasso (Targos) <targos@protonmail.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 8FCC A13F EF1D 0C2E 9100  8E09 770F 7A9A 5AE1 5600
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 20.4M  100 20.4M    0     0  4095k      0  0:00:05  0:00:05 --:--:-- 5121k
node-v12.9.1-linux-x64.tar.gz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
Authenticity of package archive can not be assured. Exiting.
```

and I am using zsh as primary shell, could you please show me how to tackle this issue? Thanks

**Comment by davidroeca** [comment]
I had the same problem. Wiping the `~/.asdf` directory a few times and trying again seemed to do the trick for me. Not an ideal solution, but it seems to be a spotty bug. Also making sure to run some of the installation steps in bash helped (not an ideal long-term solution).

**Comment by longthanhtran** [comment]
Today when I come back and try with newer version (12.10.0) and it just works fine, perhaps a bug fix has been released (on Ubuntu 18.04). I will verify again on my Mac, hopefully it also works.

**Comment by longthanhtran** [comment]
yah, it works again on my Mac also, happy to close this issue
